### PR TITLE
[SPARK-20521][DOC][CORE]The default of 'spark.worker.cleanup.appDataTtl' should be 604800 in spark-standalone.md

### PR DESCRIPTION
**Pull request by guoxiaolongzte**
## What changes were proposed in this pull request?

Currently, our project needs to be set to clean up the worker directory cleanup cycle is three days. 
When I follow http://spark.apache.org/docs/latest/spark-standalone.html, configure the 'spark.worker.cleanup.appDataTtl' parameter, I configured to 3 * 24 * 3600.
When I start the spark service, the startup fails, and the worker log displays the error log as follows:

2017-04-28 15:02:03,306 INFO Utils: Successfully started service 'sparkWorker' on port 48728.
Exception in thread "main" java.lang.NumberFormatException: For input string: "3 * 24 * 3600"
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.lang.Long.parseLong(Long.java:430)
	at java.lang.Long.parseLong(Long.java:483)
	at scala.collection.immutable.StringLike$class.toLong(StringLike.scala:276)
	at scala.collection.immutable.StringOps.toLong(StringOps.scala:29)
	at org.apache.spark.SparkConf$$anonfun$getLong$2.apply(SparkConf.scala:380)
	at org.apache.spark.SparkConf$$anonfun$getLong$2.apply(SparkConf.scala:380)
	at scala.Option.map(Option.scala:146)
	at org.apache.spark.SparkConf.getLong(SparkConf.scala:380)
	at org.apache.spark.deploy.worker.Worker.<init>(Worker.scala:100)
	at org.apache.spark.deploy.worker.Worker$.startRpcEnvAndEndpoint(Worker.scala:730)
	at org.apache.spark.deploy.worker.Worker$.main(Worker.scala:709)
	at org.apache.spark.deploy.worker.Worker.main(Worker.scala)

**Because we put 7 * 24 * 3600 as a string, forced to convert to the dragon type,  will lead to problems in the program.**

**So I think the default value of the current configuration should be a specific long value, rather than 7 * 24 * 3600,should be 604800. Because it would mislead users for similar configurations, resulting in spark start failure.**

## How was this patch tested?
manual tests

Please review http://spark.apache.org/contributing.html before opening a pull request.

### CODE REVIEW

**Comment by srowen** [comment]
Yes, it doesn't literally mean you can type this value. It's not a number. Although I think this would be generally understood, OK, easier to write the number. 

Please read http://spark.apache.org/contributing.html and don't create a JIRA for this.

**Comment by guoxiaolongzte** [comment]
So,I understand the need to configure 7 * 24 * 3600.
Is it worth changing or not worth changing?@srowen

**Comment by HyukjinKwon** [comment]
@guoxiaolongzte could you fill up the PR title?

File: docs/spark-standalone.md [hunk 1]
```diff
@@ -242,7 +242,7 @@ SPARK_WORKER_OPTS supports the following system properties:
 </tr>
 <tr>
   <td><code>spark.worker.cleanup.appDataTtl</code></td>
-  <td>7 * 24 * 3600 (7 days)</td>
+  <td>604800 (7 days)</td>
```
Comment (felixcheung) [inline docs/spark-standalone.md]: could you change this to
`604800 (7 days, 7 * 24 * 3600)`

**Review by felixcheung — COMMENTED** [review]
please update the PR title to reflect what this is

File: docs/spark-standalone.md (hunk 1 above)
Comment (guoxiaolongzte) [inline docs/spark-standalone.md]: Thank you code review, this will make the user more understanding.

**Comment by guoxiaolongzte** [comment]
@HyukjinKwon i has filled up the PR title,thank you.

**Comment by guoxiaolongzte** [comment]
@felixcheung I have changed, thank you.

**Comment by srowen** [comment]
Merged to master/2.2

**Pull request closed without merging**